README.md

```diff
@@ -27,6 +27,7 @@ machine-controller tries to follow as close as possible the Kubernetes version
 [support policy](https://kubernetes.io/docs/setup/release/version-skew-policy/).
 
 Currently supported K8S versions are:
```

Comment:
Nit: we should drop 1.15 from supported versions since we don't test it any longer.